### ISSUE TITLE
Upload the extension to OpenVSX

### ISSUE THREAD

**Issue by tani** [konn/vscode-generic-input-method#15]
Thank you for the great extension.
I am using this plugin with VSCodium to edit code for Isabelle/HOL.
The latest 2022 version of Isabelle/HOL bundles the VSCodium (not VSCode) to edit the code.
VSCodium, as you might know, could not retrieve the extension from vscode marketplace.
The editor only retrieve the extension from OpenVSX.
Please consider uploading the extension to OpenVSX.

Best regards

**Comment by RalfJung** [comment]
Yes this would be great -- currently the extension is not available for open-source users unfortunately. :(

**Comment by konn** [comment]
@tani @RalfJung Hi, thank you for your warm voices, and sorry for late respose. The situation is that this project currently has only one maintainer (i.e. me) for the time being, I am using official VSCode only and have less usage of this extension itself. Being said, I have no motivation or time resource to release this to VSCodium, unfortunately.

By the way, this project itself is intended as an OSS project indeed, so new contributors are always wellcome! If you have any inconvenience around it, you can contribute a workflow to release this also to Codium (or you can just fork)! This is how OSS work cooperatively - instead of just complaining, IMHO.

**Comment by RalfJung** [comment]
Sorry, I didn't mean to come across as just complaining. I'm contributing to a bunch of other open-source projects so I know the process well -- but like all of us, my days are finite, so I merely wanted to express that this would indeed be a great thing to have but unfortunately do not have the time to work on it myself. Obviously we're not entitled for you do do anything for us here, and I am sorry if that was the impression that my comment left. Thanks for your work on this extension!

**Comment by RalfJung** [comment]
Also, for those that are not aware (like me until yesterday) -- it is entirely possible to use this extension with vscodium, by downloading the file from [here](https://marketplace.visualstudio.com/_apis/public/gallery/publishers/mr-konn/vsextensions/generic-input-method/0.0.12/vspackage) and running the "Install from VSIX" command.

**Comment by tani** [comment]
Thank you for the response. I will ask the openvsx dev team to upload this project to OpenVSX continuously usign GH Actions. https://github.com/open-vsx/publish-extensions
